### DISCUSSION TITLE
Can I change which webcam is accessed from within hubs?

### DISCUSSION THREAD

**Discussion by CaptainCalliope** [mozilla/hubs#3058] [General Questions & Discussion] [answered]
Here's a potential issue. I want to be able to change which webcam is used (I have multiple hardware and software webcams such as snapcam, nvidia broadcast, or obs live camera), but I can't seem to find any option within Hubs to do so. I think I've set the default camera in the browser to the one I want but I'm reading that there's a bug there on the browser side (i'm using edge/chromium). An example of where this works is meet.jit.si, where I do have an option to change cameras in the app ui. So I know it's possible.

If I'm not missing something, I think there might be a browser bug here that can be worked around with a feature enhancement. What do y'all think?

**Comment by robinkwilson** [comment, marked as answer]
Currently there's no option in hubs to specify which webcam is accessed. However, you can usually select which one is on the pop up when the browser asks for permission to access your webcam. 

To work around this. Before accessing the hubs space, I'd set the webcam you want as the default on your computer, then join the hubs room. When the browser requests access to the camera, give it permissions only to the one you want. 

I agree that would be a great feature to add, if you'd like to make a hubs github issue with this as a feature request (https://github.com/mozilla/hubs/issues), that would help us!! People can vote for this via emoji's on your issue.
